src/main/java/team/creative/creativecore/common/util/type/list/SortingList.java

```diff
@@ -66,8 +62,8 @@ public void setBlacklist() {
      *            the array can either contain a Material, a Block, an Item, an
      *            ItemStack or a String for the OreDictionary. */
     public void addSortingObjects(Object... objects) {
-        for (int i = 0; i < objects.length; i++) {
-            addSortingObject(objects[i]);
+        for (Object object: objects) {
```

Comment:
Don't see a reason for this change. Many of these changes are just for the sake of it.

Reply:
indeed, there's some parts of the code using it and others no, the change is *for the sake* and for consistency. May not but is better do the change now than never

also, just to note, enhanced for is more faster used in LinkedList. ArrayList and variants performance is the same

Reply:
I do that a lot, so the original for loop the preferred way (it also works without creating another object).

Reply:
So I revert it?